_posts/2024-04-27-fcmps.md

```diff
@@ -6,7 +6,7 @@ tags: [crypto, community]
 author: Luke "Kayaba" Parker
 ---
 
-Full-Chain Membership Proofs, as a concept, is a replacement for rings within the Monero protocol. While rings have offered sender privacy to Monero since it launched, they're vulnerable to attacks such as the [EAE attack](https://www.youtube.com/watch?v=iABIcsDJKyM&list=PLsSYUeVwrHBnAUre2G_LYDsdo-tD0ov-y&index=9&pp=iAQB), [have difficulties upon chain reorganzations](https://www.youtube.com/watch?v=6CVcirD90pg&list=PLsSYUeVwrHBnAUre2G_LYDsdo-tD0ov-y&index=4&pp=iAQB), and in general enable statistical analysis (mitigated by distribution of the decoy selection algorithm). Full-Chain Membership Proofs prove the output spent is one of _any_ output on the chain, effectively removing all of these risks. This means every input goes from an immediate anonymity set of 16 to 100,000,000.
+Full-Chain Membership Proofs, as a concept, is a replacement for rings within the Monero protocol. While rings have offered sender privacy to Monero since it launched, they're vulnerable to attacks such as the [EAE attack](https://www.youtube.com/watch?v=iABIcsDJKyM&list=PLsSYUeVwrHBnAUre2G_LYDsdo-tD0ov-y&index=9&pp=iAQB), [have difficulties upon chain reorganizations](https://www.youtube.com/watch?v=6CVcirD90pg&list=PLsSYUeVwrHBnAUre2G_LYDsdo-tD0ov-y&index=4&pp=iAQB), and in general enable statistical analysis (mitigated by distribution of the decoy selection algorithm). Full-Chain Membership Proofs prove the output spent is one of _any_ output on the chain, effectively removing all of these risks. This means every input goes from an immediate anonymity set of 16 to 100,000,000.
```

Comment:
```suggestion
Full-Chain Membership Proofs [FCMP], as a concept, is a replacement for ring signatures within the Monero protocol. While rings have offered sender privacy to Monero since it launched, they're vulnerable to attacks such as the [EAE attack](https://www.youtube.com/watch?v=iABIcsDJKyM&list=PLsSYUeVwrHBnAUre2G_LYDsdo-tD0ov-y&index=9&pp=iAQB), [have difficulties upon chain reorganizations](https://www.youtube.com/watch?v=6CVcirD90pg&list=PLsSYUeVwrHBnAUre2G_LYDsdo-tD0ov-y&index=4&pp=iAQB), and in general enable statistical analysis (mitigated by distribution of the decoy selection algorithm). FCMPs prove the output spent is one of _any_ output on the chain, effectively removing all of these risks. This means every input goes from an immediate anonymity set of 16 to 100,000,000.
```

Reply:
FCMPs aren't a replacement for ring signatures, solely an alternative to ring-based membership proofs which we would be replacing with any deployment to Monero.